app/views/publishers/vacancies/build/applying_for_the_job.html.slim

```diff
@@ -23,12 +23,20 @@
             - else
                 = f.govuk_url_field :application_link, label: { size: "s" }, form_group: { classes: "optional-field" }
           - else
-            = govuk_notification_banner title: t("banners.important") do |banner|
-              - banner.add_heading text: t(".notification.heading")
-              = t(".notification.description.content_html", link: govuk_link_to(t(".notification.description.link"), page_path("job-application-preview"), target: "_blank"))
+            - if current_variant?(:"2021_04_application_preview_test", :default)
+              = govuk_notification_banner title: t("banners.important") do |banner|
+                - banner.add_heading text: t(".notification.heading")
+                = t(".notification.description.content_html", link: govuk_link_to(t(".notification.description.link"), page_path("job-application-preview"), target: "_blank"))
             = f.govuk_radio_buttons_fieldset :enable_job_applications do
-              = f.govuk_radio_button :enable_job_applications, true, link_errors: true do
```

Comment:
You might be able to do 
```ruby
= f.govuk_text_area :govuk_radio_button :enable_job_applications, true, link_errors: true, (label: { text: "A different label" } if current_variant?)
```
Or something along those lines

Reply:
Tried this and I can't seem to get it working. Just keep getting syntax errors.

If I wrap the brackets around the text instead it works when the variant being tested for (in the conditional) has been applied. It renders a blank label if it hasn't been applied, which I guess, makes sense.

Reply:
Have you tried something like this?

```
class TheController
  helper_method :enable_job_applications_true_label

  def enable_job_applications_true_label
    if current_variant?(:"2021_04_application_preview_test", :label_link)
      # something
    else
      # something else
    end
  end
end
```

```ruby
= f.govuk_radio_button :enable_job_applications, true, label: { enable_job_applications_true_label }, link_errors: true do
```

Reply:
In that case this might work nicely

```ruby
= f.govuk_radio_button :enable_job_applications, true, link_errors: true, label: { text: current_variant?... ? "this translation" : "that translation" }
```